src/util/pelz_json_parser.c

```diff
@@ -155,6 +176,40 @@ int encrypt_parser(cJSON * json, charbuf * key_id, charbuf * data)
     pelz_log(LOG_ERR, "Missing required JSON key: enc_data_len.");
     return (1);
   }
+  /* This will be re-instated when signatures and certificates are no longer optional in JSON requests
```

Comment:
For now, anyway, I don't think we want to enforce the mandatory use of signed requests. At some point, some type of enforcement is necessary to prevent the bypass of this mechanism, but I think, in line with my previous comments, this commented out code should move into the functionality that validates the signed request. If this is only called for request types which are signed, I think this removes any dependency on whether the use of signed requests is enforced or not.